client/src/pages/CreatePost.js

```diff
@@ -58,54 +113,81 @@ export default (props) => {
       >
         Create a Post
       </CustomH1>
-      <div className="settings">
-        <CustomModal
-          title={selected ? createPostSettings[selected].title : ""}
-          className="post-modal"
-          content={
+      <StyledForm onSubmit={handleSubmit}>
+        <div className="settings">
+          <CustomModal
+            title={selected ? createPostSettings[selected].title : ""}
+            className="post-modal"
+            content={
+              <RadioGroup
+                flex={true}
+                padding={"1.3rem 0"}
+                onChange={handleData(selected)}
+                options={options}
+                value={data[selected]}
+                defaultValue={data[selected]}
+              />
+            }
+            onClose={closeModal}
+            visible={modal}
+            closable={false}
+          />
+          <div className="buttons">
+            <DownArrowButton
+              handleClick={showModal(shareWith)}
+              label={data.shareWith}
+              color={theme.colors.royalBlue}
+              bgcolor={"#fff"}
+              long="true"
```

Comment:
so this is string `"true"` and not boolean?

Reply:
Yea, when I change it to `long={true}` I get this warning:
<img width="429" alt="Screen Shot 2020-04-19 at 9 40 21 AM" src="https://user-images.githubusercontent.com/42228386/79693844-dce8d700-8221-11ea-9de3-59fdcfaf17ab.png">